docs/docs/submit-to-starter-library.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Submit to Starter Library
+---
+
+Have you created a Gatsby starter you'd like to add to the [Starter Library](/starters/)? Follow these instructions.
+
+## Steps
+
+To get your site added to the starter library, follow the two steps below.
+
+1.  If this is your first contribution to the Gatsby open source repo, follow the [Contribution guidelines](/docs/how-to-contribute/#contributing-to-the-repo).
+
+2.  Edit the [`starters.yml`](https://github.com/gatsbyjs/gatsby/blob/master/docs/starters.yml) file by adding your starter information to the bottom of the list in the following format:
+
+```yaml
+- url: Link to a demo of your starter
+  repo: Link to GitHub repo
+  description: Your starter description
+  # These correspond to the category filters in the library
```

Comment:
Maybe a more specific word than "these"? At first I thought it referred to the description. Most devs probably won't unless they are new-ish and not used to commenting...which probably always comes before the line the comment is about, I'm assuming!